rest-service/manager_rest/rest/resources_v3/cluster.py

```diff
@@ -112,6 +113,7 @@ def put(self, node_id, cluster):
             'host_ip': {'type': unicode},
             'node_name': {'type': unicode},
         })
+        details.update({'database_ip': config.instance.postgresql_host})
```

Comment:
we have this here, so the additional request in the cloudify-common restclient PR isn't even needed?

Reply:
Fixed, moved validations to the joining node's side rather than the Master node's